tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{27,34,35},docs,linters,packaging,migrations
```

Comment:
Will Tox use Python 3 for packaging, docs and linters envs? Or do you have to set `basepython`?

Reply:
Nice catch!

Since the introduction of `python_requires`, Tox actually fails to run in a *vanilla* setting, e.g. running `tox -e packaging` on a system with `/usr/bin/python` pointing to Python 2.

I'll fix this by adding:
```
# ensure we run the tests with the latest supported Python version
basepython = python3.5
```
to all these Tox environments.